src/main/resources/Documentation/about.md

```diff
@@ -8,11 +8,45 @@ Parameters
 
 Project Configuration:
 
-TODO: Add Configuration parameters and instructions.
+eiffel-agen.enabled
```

Comment:
Why do you have protocol release name/version in configuration?

Reply:
Because that is how the plugin is built. In the pom.xml it has eiffel-agen, and when you edit the configurations using Gerrit UI, this is how it will look like.

Reply:
So we need to changed parameter names for every Eiffel Protocol Release?
Why can't it be named only "eiffel.enabled" without any Eiffel version names ?

Reply:
ok, discussed and solved.